recordedit/form.controller.js

```diff
@@ -392,6 +384,73 @@
             model.rows[rowIndex][column.name] = null;
         }
 
+        function openSelectAll(column) {
+            var params = {
+                column: column,
+                displayType: columnToDisplayType(column)
+            };
+
+            modalUtils.showModal({
+                animation: false,
+                controller: "RecordeditInputPopupController",
+                controllerAs: "ctrl",
+                resolve: {
+                    params: params
+                },
+                templateUrl: UriUtils.chaiseDeploymentPath() + "common/templates/inputPopup.modal.html",
+                windowClass: "modal-select-all"
+            }, function (model) {
+                if (params.displayType === "popup-select") {
+                    // set data in view model (model.rows) and submission model (model.submissionRows)
```

Comment:
Add a little explanation why you're doing this. You can briefly explain the issue that you faced when you were just setting the value without actually copying it.